docs/reference/webhooks-reference.md

```diff
@@ -1,5 +1,173 @@
 # Webhooks
 
-::: up42.webhooks.Webhooks
-    selection:
-        members: {{ funcs_webhooks }}
+The Webhooks class enables creation and modification of custom event notifications with [webhooks](webhooks.md).
+
+```python
+webhook = up42.initialize_webhook(webhook_id="1df1ebb0-78a4-55d9-b806-15d22e391bd3")
+```
+
+A webhook is a method for sending event notifications from one application to another. When something happens in a source system indicated by an event, the webhook transmits an event notification via HTTPS to a specific URL.
+
+You can set up webhooks to receive notifications in the following cases:
+
+- **Job status updates**. Every time a [job status](../../reference/job-reference/#status) changes.
+- **Order status updates**. When a new [order](../../reference/order-reference/#status) is completed — successfully or not.
+
+For more information about creating a webhook, view the UP42 [webhooks documentation](https://docs.up42.com/account/webhooks)
+
+## Webhooks
+
+### info
+
+The `info` attribute returns metadata of a specific webhook.
+
+The returned format is `dict`
+
+<h5> Example </h5>
+
+```python
+webhook.info
+```
+
+### get_webhooks()
+
+The `get_webhooks()` function returns all registered webhooks for this workspace.
+
+```python
+get_webhooks(return_json)
+```
+
+The returned format is `list[Webhook]`.
+
+<h5> Arguments </h5>
+
+| Argument      | Overview                                                                                                                                                               |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `return_json` | **bool**<br/>Determines how to return webhooks:<br/><ul><li>`True`: return JSON.</li><li>`False`: return webhook class objects.</li></ul>The default value is `False`. |
+
+<h5> Example </h5>
+
+```python
+webhooks = up42.get_webhooks(return_json=False)
```

Comment:
The functions that initialize with `up42.` are of the up42 class I'm afraid... Not of the webhook class:
- get_webhooks()
- create_webhook()
- get_webhook_events()

For the rest of them: I think we should rename the page to **Webhook**, so that there's no discrepancy in reference page naming.

Reply:
@dustgalactic I was wondering about this. Should I move them to up42-reference.md?

Reply:
Yes, please do